cmd/ddev/cmd/config.go

```diff
@@ -466,6 +470,18 @@ func handleMainConfigArgs(cmd *cobra.Command, _ []string, app *ddevapp.DdevApp)
 		app.HostDBPort = hostDBPortArg
 	}
 
+	if cmd.Flag("nfs-mount-enabled").Changed {
+		if v, _ := cmd.Flags().GetBool("nfs-mount-enabled"); v {
+			app.SetPerformanceMode(types.PerformanceModeNFS)
```

Comment:
```suggestion
			app.SetPerformanceMode(types.PerformanceModeNFS)
						dirty = true

```

Reply:
We don't have this flag in config, config-global only.